data/openai

```diff
@@ -1,2 +1,12 @@
 # Main domain
 openai.com
+ai.com
+
+# Api
+auth0.com
+sentry.io
+unpkg.com
+
+# CDN
+full:openaiapi-site.azureedge.net
+full:challenges.cloudflare.com
```

Comment:
We need more evidence to prove lines `6` `7` `8` and `12` belong to `openai`.

Reply:
They don't, openai is using these services

Reply:
They should not include in this file if they are not belonging to `openai` or are only used on `openai`.